front/src/applications/stdcmV2/components/StdcmResults.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+
+import { CheckCircle } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
+import { generateCodeNumber } from 'applications/stdcm/utils';
+import type { RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
+import { formatDateToString } from 'utils/date';
+
+import StdcmTableResults from './StdcmTableResults';
+
+type StcdmResultsV2Props = {
+  stdcmData: StdcmV2SuccessResponse;
+  pathProperties?: ManageTrainSchedulePathProperties;
+  rollingStockData: RollingStockWithLiveries;
+  speedLimitByTag?: string;
+  simulationCreationDate?: Date;
```

Comment:
You can remove the 3 props if you have moved them in stdcmData
```suggestion
```

Reply:
fix